src/site/stages/build/process-cms-exports/transformers/node-va_form.js

```diff
@@ -1,46 +1,52 @@
 const {
   createMetaTagArray,
   getDrupalValue,
+  isPublished,
   utcToEpochTime,
-  getWysiwygString,
 } = require('./helpers');
 
-const transform = entity => ({
+const transform = (entity, { ancestors }) => ({
   entityType: 'node',
   entityBundle: 'va_form',
   title: getDrupalValue(entity.title),
   created: utcToEpochTime(getDrupalValue(entity.created)),
   changed: utcToEpochTime(getDrupalValue(entity.changed)),
   entityMetatags: createMetaTagArray(entity.metatag.value),
+  entityPublished: isPublished(getDrupalValue(entity.status)),
   fieldAdministration: entity.fieldAdministration[0],
-  fieldBenefitCategories: entity.fieldBenefitCategories,
+  fieldBenefitCategories: entity.fieldBenefitCategories.map(
+    ({ fieldHomePageHubLabel }) => ({
+      entity: { fieldHomePageHubLabel },
+    }),
+  ),
   fieldVaFormAdministration: entity.fieldVaFormAdministration[0],
-  fieldVaFormIssueDate: getDrupalValue(entity.fieldVaFormIssueDate),
+  fieldVaFormIssueDate: entity.fieldVaFormIssueDate[0] || null,
   fieldVaFormLinkTeasers: entity.fieldVaFormLinkTeasers,
   fieldVaFormName: getDrupalValue(entity.fieldVaFormName),
   fieldVaFormNumber: getDrupalValue(entity.fieldVaFormNumber),
   fieldVaFormNumPages: getDrupalValue(entity.fieldVaFormNumPages),
-  fieldVaFormRelatedForms: entity.fieldVaFormRelatedForms.map(n => ({
-    entity: {
-      fieldVaFormName: n.fieldVaFormName,
-      fieldVaFormNumber: n.fieldVaFormNumber,
-      fieldVaFormUsage: n.fieldVaFormUsage,
-      fieldVaFormUrl: n.fieldVaFormUrl,
-    },
+  fieldVaFormRelatedForms: entity.fieldVaFormRelatedForms.map(e => ({
+    entity: !ancestors.find(r => r.entity.uuid === e.uuid)
+      ? {
+          fieldVaFormName: e.fieldVaFormName,
+          fieldVaFormNumber: e.fieldVaFormNumber,
+          fieldVaFormUsage: e.fieldVaFormUsage,
+          fieldVaFormUrl: e.fieldVaFormUrl,
+        }
+      : {
+          fieldVaFormName: getDrupalValue(e.field_va_form_name),
+          fieldVaFormNumber: getDrupalValue(e.field_va_form_number),
+          fieldVaFormUsage: e.field_va_form_usage[0] || null,
+          fieldVaFormUrl: e.field_va_form_url[0] || null,
+        },
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/blob/4c17d79357f4968eae0a2a0f97f15106d6e1e97b/src/site/layouts/va_form.drupal.liquid#L118-L122